README.md

```diff
@@ -450,6 +470,20 @@ want/need to set up MFA.
 The `loggerClass` parameter specifies the name of a PSR-3 compatible class that
 can be autoloaded, to use as the logger within the module.
 
+##### New "Recovery Contacts" Feature (COMING SOON)
+
+The `recoveryContacts*` parameters allow you to call an API to retrieve a list
+of recovery contacts who should be offered when the user requests help with
```

Comment:
```suggestion
of recovery contact addresses which should be offered when the user requests help with
```
Sounded strange to my ear saying that the contacts would be offered. Unless you had Romans 12:1-2 in mind. :)